.github/workflows/main.yaml

```diff
@@ -2,16 +2,19 @@ name: alchemist
 
 on: [pull_request]
 
+env:
+  FLUTTER_VERSION: 3.16.0
+
 jobs:
-  build:
+  analyze:
```

Comment:
We'll analyze against the lowest version we support. This means that contributors should be using that version (`fvm` is helpful here). We can/should probably add something that details that in our [CONTRIBUTING.md](https://github.com/Betterment/alchemist/blob/main/CONTRIBUTING.md).